pos_loyalty/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+* Go to *Point of Sale > Configuration > Loyalty Programs* and define a new loyalty program with specific rules and gifts.
+* Assign the loyalty program to the desired Point of Sale.
```

Comment:
This info should be in CONFIGURATION.rst

I also think this module is complex enough to write a more detailed usage instructions.